.github/ISSUE_TEMPLATE.md

```diff
@@ -1,18 +0,0 @@
-If you are reporting bug/issue, please provide detailed Repro instructions.
-
-## Repro Steps
-1. 
-2. 
-
-##  Actual Behavior
-
-
-## Expected Behavior
-
-
-## Version Info 
-SDK Version :
-.NET Version :	 						
-How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) :
-OS :
-Hosting Info (IIS/Azure WebApps/ etc) :
```

Comment:
Can you please put these questions in the Bug template?
We need to know these details when investigating reported issues

Reply:
Oh, i see what you did. It's in the Describe writing prompt.
I'm okay with this, but if people don't follow the prompt we will need to put these questions back.

Reply:
I've made it more explicit.